src/Sarif.Viewer.VisualStudio/CodeFlowToTreeConverter.cs

```diff
@@ -10,9 +10,9 @@ namespace Microsoft.Sarif.Viewer.VisualStudio
 {
     internal static class CodeFlowToTreeConverter
     {
-        internal static List<CallTreeNode> Convert(CodeFlow codeFlow, Run run)
+        internal static List<CallTreeNode> Convert(CodeFlow codeFlow, Run run, int resultId, int runIndex)
         {
-            var root = new CallTreeNode { Children = new List<CallTreeNode>() };
+            var root = new CallTreeNode(resultId, runIndex) { Children = new List<CallTreeNode>() };
```

Comment:
>resultId, runInde [](start = 40, length = 17)

Used named parameters here since they are both integers and can easily be transposed..